src/components/BasicForm/BasicForm.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import Select, { components } from "react-select";
+import FormItem from "../BasicForm/FormItem";
+import Button from "../Button/Button";
+import '../../style/BasicForm/basicform.css'
```

Comment:
there is import for styles but it's not used, I see you use inline styles. Please remove them and configure css modules. Css modules let's you define scoped styles to that component which prevents style collision.

Reply:
removed inline styles, create modules